types/sharedb/index.d.ts

```diff
@@ -4,7 +4,7 @@
 //                 Eric Hwang <https://github.com/ericyhwang>
 //                 Peter Xu <https://github.com/pxpeterxu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+// TypeScript Version: 3.7
```

Comment:
Did you intend to modify the TypeScript Version indicator here?

This PR has no other changes in the `sharedb` types, and I don't believe the types use any language features that require 3.7.

Reply:
From memory, running `npm test` required it? It looks like the tests for `sharedb` include QUnit as a reference

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2a02c20372ad2dbc512b4ff25864339b9020c259/types/sharedb/sharedb-tests.ts#L1

Reply:
Oh, I somehow missed that when I was looking at the file.

I don't think that file actually uses qunit in any way. May have been a copy-paste by whoever wrote that file originally. The next time someone modifies the file, I'll ask them to remove that reference.